packages/fe-fpm-writer/templates/common/partials/AdditionalDependencies.xml

```diff
@@ -0,0 +1 @@
+<%- typeof dependencies !== 'undefined' ? Object.entries(dependencies).map((dependency) => ` ${dependency[0]}="${dependency[1]}"`).join('') : '' %>
```

Comment:
That seems like an odd place or way to do it. If we need to reuse it, couldn't we do this already in the (debuggable) code and then give the result as part of the config to the template?

Reply:
@tobiasqueck 
thanks for pointing.
Some background:
1. map with additional dependencies was added some time ago and currently used by custom sections only to include `sap.fe.macros` based on ui5 version - https://github.com/SAP/open-ux-tools/blob/feat/1060/containerForSectionFragment/packages/fe-fpm-writer/src/section/index.ts#L41;
2. Idea was that probably in future similar will be added to custom views;

I simplified logic within commit d2a3c6b07ea34eb907b8b8cd5c89eae108ad6334
1. there is no more `partials`
2. instead of map - directly string is used